lib/stack_master/parameter_resolver.rb

```diff
@@ -16,7 +16,12 @@ def initialize(config, stack_definition, parameters)
 
     def resolve
       @parameters.reduce({}) do |parameters, (key, value)|
-        parameters[key] = resolve_parameter_value(value)
+        begin
+          parameters[key] = resolve_parameter_value(value)
+        rescue InvalidParameter
+          say "Unable to resolve parameter #{key.inspect} value causing error: #{$!.message}"
```

Comment:
Actually I'm thinking it would be better to raise an error with a good
error message and let it bubble to the top.